GPUVerifyVCGen/UniformityAnalyser.cs

```diff
@@ -0,0 +1,579 @@
+﻿namespace GPUVerify

```

Comment:
Does this file need the usual banner we have on all other files?

Reply:
Done.